docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -338,6 +338,21 @@ ctr -n k8s.io t exec -t --exec-id etcd ${ETCD_CONTAINER_ID} etcdctl \
 Follow the VM restore process in provider-specific section.
 * [Restore VM for machine in vSphere]({{< relref "#restore-vm-for-machine-in-vsphere" >}})
 
+#### Nodes cycling due to insufficient pod cidrs
+
+Kuberenetes allocates a cidr block per node from within `clusterNetwork.pods.cidrBlocks` with a default cidr mask size of /24. This node cidr mask size can be adjusted using the [optional cidrMaskSize field]({{< relref "../getting-started/optional/cni/#node-ips-configuration-option" >}}). If more nodes are created than the `clusterNetwork.pods.cidrBlocks` can accomodate, new nodes will will not receive a cidr block for pods. 
```

Comment:
```suggestion
Kuberenetes allocates a cidr block per node from within `clusterNetwork.pods.cidrBlocks` with a default cidr mask size of /24. This node cidr mask size can be adjusted using the [optional cidrMaskSize field]({{< relref "../getting-started/optional/cni/#node-ips-configuration-option" >}}). If more nodes are created than the `clusterNetwork.pods.cidrBlocks` can accommodate, new nodes will not receive a cidr block for pods. 
```

Reply:
> If more nodes are created than the `clusterNetwork.pods.cidrBlocks` can `accomodate`, new nodes will will not receive a cidr block for pods.

what does this sentence mean exactly? how do we know how many nodes can pods.cidrBlocks accommodate? 
what does it mean by new nodes will not receive a cidr blocks for pods?

Reply:
Updated with an example of podblock and node mask and clarified that kube-controller-manager will not provide the cidr block to the node.